Sources/NIOSSL/SSLCertificateName.swift

```diff
@@ -0,0 +1,81 @@
+@_implementationOnly import CNIOBoringSSL
+
+public struct SSLCertificateNameType: Equatable, Hashable, Sendable {
+    public var nid: Int32
```

Comment:
I suspect this can be `internal`: the values are an implementation detail of BoringSSL and not particularly meaningful for anyone else.

Reply:
Oh sorry, I didn't mean the whole type, just the `nid` property.